### DISCUSSION TITLE
2024/01/28/acm/codeforces/CodeTONRound7/

### DISCUSSION THREAD

**Discussion by giscus** [Hukeqing/Hukeqing.github.io#13] [General(讨论贴)]
# 2024/01/28/acm/codeforces/CodeTONRound7/

CodeTON Round 7 (Div. 1 + Div. 2, Rated, Prizes!) 个人写题记录

https://blog.mauve.icu/2024/01/28/acm/codeforces/CodeTONRound7/

**Comment by kernelbin** [comment]
教教我打Div1!
